src/Parse/ParseQuery.php

```diff
@@ -623,6 +623,26 @@ public function withinGeoBox($key, $southwest, $northeast)
     }
 
     /**
+     * Add a constraint to the query that requires a particular key's
+     * coordinates be contained within a given polygon
+     *
+     * @param string        $key       The key of the ParseGeoPoint
+     * @param array         $points    Array of ParseGeoPoint
+     *
+     * @return ParseQuery Returns this query, so you can chain this call.
+     */
+    public function withinPolygon($key, $points)
+    {
+        $this->addCondition(
+            $key,
+            '$geoWithin',
+            ['$polygon' => $points]
+        );
+
+        return $this;
+    }
+
```

Comment:
Nice! I like the cases, looks pretty good to me at the moment. Also noticed that `withinPolygon` pushed out in a release about 3 days ago. However some of these will fail until your most recent changes are incorporated into the next release. 

In the interest of keeping in step with what's currently released we'll hold onto this PR for now. I suspect we should see another release in a week or two and then we'll move this along.